docs/commands/GetArchiveDaysMC.md

```diff
@@ -0,0 +1,133 @@
+# GetArchiveDaysMC
+
+Command to request/receive day pulse counter's values from device archive.
+
+
+## Request
+
+### Format
+
+| Size       | Type                                                         | Field                                      |
+| ---------- | ------------------------------------------------------------ | ------------------------------------------ |
+| `1`        | `byte`                                                       | command id = `0x1b`                        |
+| `1`        | `byte`                                                       | command size = `4`                         |
+| `2`        | [packed date](../types.md#packed-date)                       | [start date](#date)                        |
+| `1..5`     | [extended value](../types.md#extended-value)                 | [channels bit set](#channels-bit-set)      |
```

Comment:
```suggestion
| `1..5`     | [extended value](../types.md#extended-value)                 | [channels bit set](../../types.md#channels-bit-set) |
```

this was approved just today and will be fixed everywhere in the next pr

Reply:
fixed

Reply:
is it commited?

Reply:
is it commited?

Reply:
and for Response

Reply:
fixed